radis/test/io/test_hitemp.py

```diff
@@ -281,6 +281,34 @@ def test_fetch_hitemp_all_molecules(molecule, verbose=True, *args, **kwargs):
     assert len(df) == Nlines
 
 
+@pytest.mark.needs_connection
+@pytest.mark.download_large_databases
```

Comment:
We usually don't trigger these tests because they cost too much time on Travis. Could you write a simple test testing if we can fetch the 2010 version of OH (probably the lightest database)?